src/Sylius/Bundle/TaxonomyBundle/Doctrine/ORM/TaxonRepository.php

```diff
@@ -23,20 +22,30 @@ class TaxonRepository extends EntityRepository implements TaxonRepositoryInterfa
     /**
      * {@inheritdoc}
      */
-    public function findChildren($parentCode)
+    public function findChildren($parentCode, $isParentRoot = false)
```

Comment:
`findChildren('CODE', true)` shouldn't return grandchildren etc.
It should be rather `findChildren($parentCode)` and `findDescendants($rootCode)`.